pkg/operator/ceph/object/user/controller.go

```diff
@@ -339,17 +339,22 @@ func generateUserConfig(user *cephv1.CephObjectStoreUser) object.ObjectUser {
 	return userConfig
 }
 
+func generateCephUserSecretName(u *cephv1.CephObjectStoreUser) map[string]string {
+	m := make(map[string]string)
+	m["secretName"] = fmt.Sprintf("rook-ceph-object-user-%s-%s", u.Spec.Store, u.Name)
+	return m
+}
+
 func (r *ReconcileObjectStoreUser) generateCephUserSecret(u *cephv1.CephObjectStoreUser) *v1.Secret {
 	// Store the keys in a secret
 	secrets := map[string]string{
 		"AccessKey": *r.userConfig.AccessKey,
 		"SecretKey": *r.userConfig.SecretKey,
 	}
-
-	secretName := fmt.Sprintf("rook-ceph-object-user-%s-%s", u.Spec.Store, u.Name)
+	u.Status.Info = generateCephUserSecretName(u)
 	secret := &v1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      secretName,
+			Name:      u.Status.Info["secretName"],
```

Comment:
This will be cleaner if `generateCephUserSecretName()`just returns the secret name instead of the map.

Reply:
```suggestion
			Name:      generateCephUserSecretName(u),
```